src/Request.php

```diff
@@ -235,7 +255,7 @@ public function getId()
      * @param  string $version
      * @return self
      */
-    public function setVersion($version)
+    public function setVersion(string $version) : self
     {
         if ('2.0' == $version) {
```

Comment:
String comparison here as we have now param type declaration.

For another PR: we tend to avoid yoda conditions.

Reply:
Strict comparison: Change will be pushed shortly.

Yoda conditions: Noted. I will open an issue.